.travis.yml

```diff
@@ -32,6 +33,13 @@ matrix:
           - tox --installpkg ./dist/djangorestframework-*.whl
           - tox  # test sdist
 
+      - python: "3.6"
+        env: TOXENV=dist-optionals
+        script:
+          - python setup.py bdist_wheel
+          - tox --installpkg ./dist/djangorestframework-*.whl
+          - tox  # test sdist
+
```

Comment:
These changes don't immediately appear connected to the rest of the PR - let's drop them.

Reply:
They are necessary to cover the change in the tests, but I can move that into a separate PR.

Reply:
=> https://github.com/encode/django-rest-framework/pull/6041